vue-components/src/components/Dialog.vue

```diff
@@ -188,10 +191,17 @@ export default Vue.extend( {
 			this.$nextTick( this._resetScroll );
 		},
 		dismiss(): void {
+			/** 
+			 * Closes / dismisses the dialog
+			 */
 			this.$emit( 'dismissed' );
 			this.hide();
 		},
 		_dispatch( namespace: string ): void {
+			/**
```

Comment:
Prob needs a better explanation. I don't know how to write it because i am not sure I even understand how it works 100%.